src/Microsoft.DotNet.Interactive.SqlServer/MsSqlKernel.cs

```diff
@@ -181,6 +181,10 @@ public async Task HandleAsync(SubmitCode command, KernelInvocationContext contex
                         context.Fail(message: messageParams.Message.Message);
                         completion.SetResult(true);
                     }
+                    else
+                    {
+                        context.Display(messageParams.Message.Message.ToString());
```

Comment:
Does this branch also need a `completion.SetResult(true);`?

Reply:
Completion in this case would be handled at the end of processing the query results. We set the completion above because it's a failure case.